backend/dal/Helpers/Extensions/ParcelExtensions.cs

```diff
@@ -57,13 +57,20 @@ public static IQueryable<Entity.Parcel> GenerateQuery(this PimsContext context,
             }
 
             if (filter.NELatitude.HasValue && filter.NELongitude.HasValue && filter.SWLatitude.HasValue && filter.SWLongitude.HasValue)
-                query = query.Where(p =>
-                    p.Latitude != 0 &&
-                    p.Longitude != 0 &&
-                    p.Latitude <= filter.NELatitude &&
-                    p.Latitude >= filter.SWLatitude &&
-                    p.Longitude <= filter.NELongitude &&
-                    p.Longitude >= filter.SWLongitude);
+            {
+                var pfactory = new NetTopologySuite.Geometries.GeometryFactory();
+                var ring = new NetTopologySuite.Geometries.LinearRing(
```

Comment:
Need to make a helper extension function to do this.  Essentially it provides a way to query the db for all properties that fall within the polygon.  However I think this isn't working correctly, as all properties are not being returned when they should be.